Utils/old_content_branch.py

```diff
@@ -0,0 +1,169 @@
+
+import argparse
+import os
+import click
+import yaml
+import io
+import json
+from pkg_resources import parse_version
+
+
+def get_file(method, file_path, type_of_file):
+    data_dictionary = None
+    with open(os.path.expanduser(file_path), mode="r", encoding="utf8") as f:
+        if file_path.endswith(type_of_file):
+            read_file = f.read()
+            replaced = read_file.replace("simple: =", "simple: '='")
+            # revert str to stream for loader
+            stream = io.StringIO(replaced)
+            try:
+                data_dictionary = method(stream)
+            except Exception as e:
+                click.secho("{} has a structure issue of file type{}. Error was: {}".format(file_path, type_of_file, str(e)), fg="red")
+                return {}
+    if type(data_dictionary) is dict:
+        return data_dictionary
+    return {}
+
+
+def get_yaml(file_path):
+    return get_file(yaml.safe_load, file_path, ('yml', 'yaml'))
+
+
+def get_json(file_path):
+    return get_file(json.load, file_path, 'json')
+
+
+def handle_yml_file(file_path, new_to_version, should_rewrite=True):
```

Comment:
same here as the json one

Reply:
will change in the same way I changed the json